Documentation/Creating-your-templates.md

```diff
@@ -8,16 +8,16 @@ When you invoke SwiftGen, you can specify templates by name or by path.
 
 ### Using a full path
 
-If you use the `--templatePath` (or `-p`) option, you'll need to specify the **full path** to the template you want to use. This allows you to store your templates anywhere you want and name them anyhow you want, but can become quite annoying to type.
+If you use the `templatePath` configuration option, you'll need to specify the **relative path** to the template you want to use. This allows you to store your templates anywhere you want and name them anyhow you want, but can become quite annoying to type.
```

Comment:
The "but can become quite annoying to type" isn't really true anymore, it was referring to when you had to type it every time in the CLI, and to advocate for named templates instead (either bundled or in AppSupport).

Nowadays:
 - as we encourage using the config file, the "annoying to type" doesn't make any sense anymore
 - using a custom template relative path is quite a valid option, no need to "shame" that possibility anymore
 - pushing to store custom templates in Application Support (which is kinda why that sentence was worded that way I guess?) is not recommended anymore (as it's totally incompatible with working in teams via git). I actually wondered multiple times if there really are users using that AppSupport feature and if we shouldn't remove that possibility completely from SwiftGen, but figured "since it's already there and works, why not keep it"

Reply:
Suggestion:

> This allows you to store your templates anywhere you want (typically in the same folder/git repo of the project in need for those custom templates for example).